loom/common/assets/assets.cpp

```diff
@@ -842,7 +873,8 @@ void loom_asset_pump()
       else
       {
         // Instate the asset.
-        asset->instate(type, assetBits, dtor);        
+        asset->instate(type, assetBits, dtor);
+        asset->blob->length = size;
```

Comment:
So we can display the loaded size later on, that field wasn't used / set before.